docker-compose.yml

```diff
@@ -1,39 +1,91 @@
-nginx:
-  image: prooph/nginx:www
-  ports:
+version: '2'
+services:
+
+  mongodb:
+    image: mongo
+    ports:
+      - 27017:27017
+
+  mysql:
+    image: mysql
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=dev
+      - MYSQL_USER=dev
+      - MYSQL_PASSWORD=dev
+      - MYSQL_DATABASE=todo
+    volumes:
+      - ./vendor/prooph/pdo-event-store/scripts/mysql:/docker-entrypoint-initdb.d
+
+  nginx:
+    image: prooph/nginx:www
+    ports:
       - 8080:80
       - 443:443
       - 10081:10081
       - 10082:10082
-  links:
-    - php:php
-  volumes_from:
-    - dataphp
-
-php:
-  image: prooph/php:5.6-fpm
-  links:
-    - mariadb:mariadb
-    - mongodb:mongodb
-  volumes_from:
-    - dataphp
-
-dataphp:
-  image: debian:jessie
-  volumes:
-    - .:/var/www
-
-mariadb:
-  image: mariadb
-  ports:
-    - 3306:3306
-  environment:
-    - MYSQL_ROOT_PASSWORD=dev
-    - MYSQL_USER=dev
-    - MYSQL_PASSWORD=dev
-    - MYSQL_DATABASE=proophessor
-
-mongodb:
-    image: mongo
-    ports:
-      - 27017:27017
+    links:
+      - php:php
+    volumes:
+      - .:/var/www
+
+  php:
+    image: prooph/php:7.1-fpm
+    volumes:
+      - .:/var/www
+    depends_on:
+      - mysql
+      - mongodb
```

Comment:
you have removed mongodb-snapshot-store from the manual, so why is mongo still needed here?